supabase/migrations/20230606180645_add_bookmarked_projects.sql

```diff
@@ -0,0 +1,24 @@
+create table public.user_bookmarks (
+    "project" text not null,
+    "user_id" uuid not null,
```

Comment:
same with this, i think we could/should foriegn key this to users table

```
  user_id uuid not null references public.users(id),
```

Reply:
edit: i see this is happening below: https://github.com/jbx-protocol/juice-interface/pull/3673/files#diff-2a0597f9a314f672eebbf92f1be5442b2a5a8f757b530aefde412d854cd5ffccR22

Might be clearer to bundle all the changes up into the single `create table`, but either works i guess!

Reply:
yeah for sure. i had quickly copied much of this over from previous migrations to make sure i was getting syntax right, and forgot to go back over this for some polish